ios/engine/KMEI/KeymanEngine/Classes/Settings/InstalledLanguagesViewController.swift

```diff
@@ -173,7 +173,9 @@ public class InstalledLanguagesViewController: UITableViewController, UIAlertVie
       if let langName = keyboardLanguages[l]?.name {
         log.info("keyboard language \(l) \(langName) has lexical model")
```

Comment:
Do you want to breadcrumbAndLog here?

Reply:
I felt that the two spots with this particular check were a bit 'noisy' and not particularly informative, so I skipped 'em.